js/node/src/session_options_helper.cc

```diff
@@ -21,6 +21,9 @@
 #ifdef USE_COREML
 #include "core/providers/coreml/coreml_provider_factory.h"
 #endif
+#ifdef USE_QNN
+//#include "core/providers/qnn/qnn_provider_factory.h"
```

Comment:
this is not required, we use the general one AppendExecutionProvider which is in API header file.

Reply:
Removed per request